plugin/napari_lattice/_reader.py

```diff
@@ -138,3 +141,24 @@ def bdv_h5_reader(path):
 
     layer_type = "image"  # optional, default is "image"
     return [(images, add_kwargs, layer_type)]
+
+
+def tiff_reader(path):

```

Comment:
```suggestion
def tiff_reader(path: ImageLike) -> Tuple[AICSImage, dict, str]:
```
Will need to import `ImageLike` from `aicsimageio.types` and `Tuple` from `typing` at the top, too.